src/main/java/com/intellij/compiler/notNullVerification/ThrowOnNullMethodVisitor.java

```diff
@@ -64,6 +65,14 @@ private void setInstrumented() {
         instrumented = true;
     }
 
+    /** This will be invoked only when visiting bytecode produced by java 8+ compiler with '-parameters' option. */
+    @Override
+    public void visitParameter(String name, int access) {
+        if (parameterNames == null) parameterNames = new ArrayList<>(argumentTypes.length);
```

Comment:
I usually use braces on everything, so I like it. I might have missed a few places in this codebase been trying to clean it up as I add new features.